packages/emitter-framework/src/csharp/components/json-converter/json-converter-resolver.tsx

```diff
@@ -0,0 +1,105 @@
+import { useTsp } from "#core/index.js";
+import { createContext, refkey, useContext, type Children, type Refkey } from "@alloy-js/core";
+import {
+  getTypeName,
+  type DurationKnownEncoding,
+  type EncodeData,
+  type Type,
+} from "@typespec/compiler";
+import { getNullableUnionInnerType } from "../utils/nullable-util.js";
+import { TimeSpanIso8601JsonConverter, TimeSpanSecondsJsonConverter } from "./json-converter.jsx";
+
+interface JsonConverterInfo {
+  refkey: Refkey;
+  name: string;
+  converter: Children;
```

Comment:
is there a way to retrieve the refkey and name from the generated Child? These info is useful when other places need to refer to the converter or use the name (i.e. for the filename)